init.go

```diff
@@ -2,56 +2,70 @@
 
 package openssl
 
-/*
-int go_openssl_version_major(void* handle);
-int go_openssl_version_minor(void* handle);
-int go_openssl_version_patch(void* handle);
-*/
 import "C"
-import (
-	"errors"
-)
+import "errors"
 
 // opensslInit loads and initialize OpenSSL.
 // If successful, it returns the major and minor OpenSSL version
 // as reported by the OpenSSL API.
 //
 // See Init() for details about file.
-func opensslInit(file string) (major, minor, patch uint, err error) {
+func opensslInit(file string, initOnly bool) error {
```

Comment:
This naming doesn't seem ideal... based on the name, `opensslInit` clearly does init, but then what exactly does it mean to "only" init? The "extra" things it can optionally do are load some additional funcs and call `go_openssl_OPENSSL_init` and `go_openssl_OPENSSL_init_crypto`--which seem like initialization. 😄 

(Whenever I see a bool passed to a func I also question whether it should actually be two functions that share some logic to make call sites clear.)

Maybe a purpose-focused name? `opensslCapabilityCheckInit` vs. `opensslInit`?
Or `opensslInitShallow` vs. `opensslInit` to keep it generic?
(IMO at minimum, details in the doc comment.)

Reply:
Agree and done 🐱.